docs/sources/flow/reference/components/prometheus.exporter.kafka.md

```diff
@@ -0,0 +1,115 @@
+| `use_tls`                   | `bool`          | Connect using TLS.                                                                                                                                                                  |         | no       |
+| `ca_file`                   | `string`        | The optional certificate authority file for TLS client authentication.                                                                                                              |         | no       |
+| `cert_file`                 | `string`        | The optional certificate file for TLS client authentication.                                                                                                                        |         | no       |
+| `key_file`                  | `string`        | The optional key file for TLS client authentication.                                                                                                                                |         | no       |
+| `insecure_skip_verify`      | `bool`          | If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure.                                                                 |         | no       |
+| `kafka_version`             | `string`        | Kafka broker version.                                                                                                                                                               | `2.0.0` | no       |
+| `use_zookeeper_lag`         | `bool`          | if you need to use a group from zookeeper.                                                                                                                                          |         | no       |
+| `zookeeper_uris`            | `array(string)` | Address array (hosts) of zookeeper server..                                                                                                                                         |         | no       |
+| `kafka_cluster_name`        | `string`        | Kafka cluster name.                                                                                                                                                                 |         | no       |
+| `metadata_refresh_interval` | `duration`      | Metadata refresh interval.                                                                                                                                                          | `1m`    | no       |
+| `allow_concurrency`         | `bool`          | If true, all scrapes will trigger kafka operations otherwise, they will share results. WARN: This should be disabled on large clusters.                                             | `true`  | no       |
```

Comment:
```suggestion
| `allow_concurrency`         | `bool`          | If set to true, all scrapes trigger Kafka operations. Otherwise, they will share results. WARNING: Disable this on large clusters.                                            | `true`  | no       |
```